Automattic/woocommerce-payments
# Add/e2e shopper subscriptions change pay method

Closes https://github.com/automattic/woocommerce-payments/issues/2223 and https://github.com/automattic/woocommerce-payments/issues/2224

#### Changes proposed in this Pull Request

Title: This PR adds 2 e2e tests that cover the Critical Flows "Shopper (subscriptions) > Change payment method to new card" and "Shopper (subscriptions) > Change payment method to saved card"

Description: This PR adds 2 e2e tests that cover these 2 critical flows:
- [Change payment method to new card](https://github.com/Automattic/woocommerce-payments/wiki/Testing-instructions-for-critical-flows#change-payment-method-to-new-card)
- [Change payment method to saved card](https://github.com/Automattic/woocommerce-payments/wiki/Testing-instructions-for-critical-flows#change-payment-method-to-saved-card)

#### Testing instructions

1. Clone this branch, and launch the e2e environment as usual.
2. Run the newly added test suite:
    ```
    npm run test:e2e tests/e2e/specs/shopper/shopper-subscriptions-change-payment-method.spec.js
    ```
3. These 2 tests should pass:
    <img width="654" alt="Screen Shot 2021-09-07 at 2 51 56 PM" src="https://user-images.githubusercontent.com/4509348/132298110-6f97c768-1f24-4b39-b65c-1ffb4de93ff1.png">
